apps/build_plant_metabolic_model/display.yaml

```diff
@@ -3,9 +3,9 @@
 #
 name     : Build Plant Metabolic Model
 subtitle : |
-    Build a metabolic model for a plant based on annotated sequences of transcripts.
+    Build a metabolic model for a plant based on annotated sequences of coding sequences.
```

Comment:
Can you rephrase?

Remember this is an app, so it covers both annotation and metabolic modeling. I thought that the passive voice implied that annotation would already be done when running the app.